src/common/metrics/case_density.tsx

```diff
@@ -71,8 +71,9 @@ function renderStatus(projections: Projections): React.ReactElement {
   ) {
     return (
       <Fragment>
-        Not enough case data is available to generate{' '}
-        {CaseIncidenceMetric.extendedMetricName.toLowerCase()}.
+        Unable to generate{' '}
+        {CaseIncidenceMetric.extendedMetricName.toLowerCase()}. This could be
+        due to insufficient data.
```

Comment:
I bet this is out of scope for now, but it would be really great to link to an FAQ or explanation about why there could be insufficient data

Reply:
Yeah...  I thought about making this more descriptive, but a FAQ or something would be great.